pkg/operation/botanist/health_check.go

```diff
@@ -889,7 +889,7 @@ func isUnstableOperationType(lastOperationType gardencorev1alpha1.LastOperationT
 // in create or delete state.
 func (b *Botanist) pardonCondition(condition gardencorev1alpha1.Condition) gardencorev1alpha1.Condition {
 	shoot := b.Shoot.Info
-	if shoot.Status.LastError != nil {
+	if len(shoot.Status.LastErrors) > 0 {
```

Comment:
Shouldn't we also keep checking for `shoot.Status.LastError` here?

Reply:
Yep, missed it here.